_config.yml

```diff
@@ -10,7 +10,9 @@ branch: 1.x
 #   {{site.webdev}}/downloads
 
 dartlang: ""
+dartlang_2: "https://dartlang-org-dev.firebaseapp.com"
```

Comment:
FYI, I've been using the following YAML names in the webdev config:

- `url` is the main URL to this site. (It's defined on line 4 above.)
- `dev-url` for the URL to the dev version of the site.
- `prev-url` for the URL to the previous version of the site (which will make sense only once Dart 2 lands on master).

Of course, we don't have to stick to these names, but I thought that I'd mention it.

Reply:
I wanted to avoid renaming this once 1.x is no longer the default site. Maybe dartlang-2?

Reply:
I've done this rename, just in case.